README.md

```diff
@@ -22,7 +22,7 @@ This container can run without any configuration on Saagie's platform.
 
 If you want to run it with an in-memory Spark, just run:
 ```
-docker run -it --rm --name zeppelin -p 8080:8080 saagie/zeppelin:sp1.12.1-mesos1.3.1-spark2.1.0
+docker run -it --rm --name zeppelin -p 8080:8080 saagie/zeppelin:sp1.12.1-mesos1.3.1-spark2.3.4
```

Comment:
SPARK_VERSION is 2.4.5 by default, maybe your example could use 2.4.5 as well?

Reply:
OK ! This was changed in December when upgrading 2.3.4 but I forgot to change it for this other upgrading.